### PR TITLE
Update UI

### PR DESCRIPTION
**Pull request by Wgis**
New look for UI
https://i.imgur.com/X9JAgEr.png (AutoSize by config pocket item count)

### CODE REVIEW

File: entities/weapons/pocket/cl_menu.lua [hunk 1]
```diff
@@ -98,6 +109,12 @@ function reload()
         table.insert(items, icon)
         i = i + 1
     end
+ 	if itemCount < GAMEMODE.Config.pocketitems then
```
Comment (Kefta) [inline entities/weapons/pocket/cl_menu.lua]: Still got tabs here

File: entities/weapons/pocket/cl_menu.lua [hunk 2]
```diff
@@ -32,14 +32,26 @@ function DarkRP.openPocketMenu()
...
-    frame = vgui.Create("DFrame")
-
+    frame = vgui.Create( "DFrame" )
+    local count = GAMEMODE.Config.pocketitems or GM.Config.pocketitems
+    frame:SetSize( 345, 32+64*(count/5)+3*(count/5))
     frame:SetTitle(DarkRP.getPhrase("drop_item"))
     frame.btnMaxim:SetVisible(false)
-    frame.btnMinim:SetVisible(false)
-    frame:SetVisible(true)
+    frame.btnMinim:SetVisible(false)
```
Comment (Kefta) [inline entities/weapons/pocket/cl_menu.lua]: Random trailing tab

File: entities/weapons/pocket/cl_menu.lua [hunk 3]
```diff
@@ -32,14 +32,26 @@ function DarkRP.openPocketMenu()
     if not wep:IsValid() or wep:GetClass() ~= "pocket" then return end
     if not pocket then pocket = {} return end
     if table.Count(pocket) == 0 then return end
-    frame = vgui.Create("DFrame")
-
+    frame = vgui.Create( "DFrame" )
```
Comment (Kefta) [inline entities/weapons/pocket/cl_menu.lua]: The DarkRP codebase consistently doesn't use airy coding (no space between parentheses and args)

**Comment by Wgis** [comment]
I'm noob in git, i simply use ctrl+v and ctrl+c

**Pull request merged**